src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -705,6 +718,8 @@ protected internal virtual void AddResponseFileCommands(CommandLineBuilderExtens
             commandLine.AppendPlusOrMinusSwitch("/optimize", _store, nameof(Optimize));
             commandLine.AppendSwitchIfNotNull("/pathmap:", PathMap);
             commandLine.AppendSwitchIfNotNull("/out:", OutputAssembly);
+            commandLine.AppendSwitchIfNotNull("/outref:", OutputRefAssembly);
```

Comment:
Definitely. That was a bug.

---
In reply to: [106561048](https://github.com/dotnet/roslyn/pull/17558#discussion_r106561048) [](ancestors = 106561048)